schemas/fingerprint-server-api.yaml

```diff
@@ -699,9 +697,57 @@ components:
                   description: >
                     Android specific emulator detection. There are 2 values:
                     • `true` - Emulated environment detected (e.g. launch inside of AVD)
-                    • `false` - No signs of emulated environment detected
-
-                    Available only for events from Android client. The field will not be present for a browser or iOS event.
+                    • `false` - No signs of emulated environment detected or the client isn't Android.
+                  example: false
+            clonedApp:
+              title: WebhookSignalResponseClonedApp
+              type: object
+              properties:
+                result:
+                  type: boolean
+                  description: >
+                    Android specific cloned application detection. There are 2 values:
+                    • `true` - Cloned application detected (e.g. fully cloned application with a slightly different package name)
```

Comment:
I would change this line to -> _Presence of app cloners work detected (e.g. fully cloned application found or launch of it inside of a not main working profile detected)_

With that rephrasing it's slightly more correct what we're doing. Also I would omit details with slightly different package name since it's an internal technique for detecting which is IP. 

Other than that everything is super cool! 💪

Reply:
done